api/server/sdk/volume_node_ops.go

```diff
@@ -93,11 +153,21 @@ func (s *VolumeServer) Unmount(
 		return nil, status.Error(codes.InvalidArgument, "Invalid Mount Path")
 	}
 
-	err := s.driver.Unmount(req.GetVolumeId(), req.GetMountPath(), req.GetOptions())
+	options := make(map[string]string)
+	options[mountattachoptions.OptionsWaitBeforeDelete] = "true"
```

Comment:
I think by default we should set OptionsDeleteAfterUnmount to true as well and update the docs accordingly.
Also it does not make sense to set WaitBeforeDelete to true if DeleteAfterUnmount was set to false.